app/controllers/api/api_controller.rb

```diff
@@ -13,6 +13,7 @@ class ApiController < ActionController::API
     rescue_from ActiveRecord::RecordInvalid, with: :invalid_transition
     rescue_from Api::Errors::InvalidTransitionError, with: :invalid_transition
     rescue_from Api::Errors::InvalidDatetimeError, with: :invalid_updated_since_response
+    rescue_from Api::Errors::RecordNotFoundError, with: :not_found_response
```

Comment:
chatted in person: although we need this eventually, we're gonna postpone till we have the correct error messaging, good catch though!